docs/getting-started.qmd

```diff
@@ -66,40 +71,63 @@ Please see below how to configure the spftware to connect to your iRODS instance
 
 ### iBridges config.json
 
-*iBridges* will create its own configuration file in `~/.ibridges/` containing the name of the last *iRODS* environment file used.  This `config.json` file can be updated to control other aspects of *iBridges*.  For example:
+*iBridges* will create its own configuration file in `~/.ibridges/ibridges_config.json` containing the name of the last *iRODS* environment file used.  This `ibridges_config.json` file can be updated to control other aspects of *iBridges*.  For example:
 
 ```json
 {
     "last_ienv": "irods_environment.json",
     "davrods_server": "https://server.fqdn.nl",
+    "eln_token": "<api key>",
+    "amber_token": "<api key>"
     "ui_tabs": [
         "tabUpDownload",
         "tabELNData",
+        "tabAmberWorkflow"
         "tabDataBundle",
         "tabCreateTicket"
     ],
-    "force_unknown_free_space": false
+    "force_transfers": false,
+    "check_free_space": false
 }
 ```
 Options:
 
-- `davrods_server`: for annotation of eLabJournal data
+- `davrods_server`: the address of a [webdav server](https://github.com/UtrechtUniversity/davrods), used to create entries in [ElabJournal](https://www.elabnext.com/products/elabjournal/)
+- `eln_token`, `amber_token`: API keys for `tabELNData` or `tabAmberWorkflow` respectively
 - `ui_tabs`: configure which tabs are shown (Browser and Info tabs always are)
     - `tabUpDownload`: a two-pane upload/download tab
     - `tabELNData`: for the Electronic Lab Notebook, eLabJournal
+    - `tabAmberWorkflow`: up and download data to and from [Amberscript](https://www.amberscript.com/en/) directly from iRODS.
     - `tabDataBundle`: (un)bundle datasets from/to four supported formats
     - `tabCreateTicket`: create iRODS tickets for anonymous access
-- `force_unknown_free_space`: ignore if resources' free space is unannotated
+- `force_transfers`: always write data to iRODS
+- `check_free_space`: checks the metadata *free_space* of resources and only shows resources with a *free_space* larger than 0.
+
+The `check_free_space` and `force_transfers` options are *REQUIRED*. The default values are  
```

Comment:
If the user fails to add them, they will be added accompanied by a warning in the logs.